### PR TITLE
workflows: update pr-auditor workflow

### PR DESCRIPTION
**Pull request by BolajiOlajide**
This pull request updates the pr-auditor workflow.

## Test plan

We have tested the code itself in https://github.com/sourcegraph/devx-service/tree/main/cmd/pr-auditor,
and the pr-auditor action should continue working as expected on this pull request.

[_Created by Sourcegraph batch change `bolaji.olajide/PR-Auditor-2.0`._](https://sourcegraph.sourcegraph.com/users/bolaji.olajide/batch-changes/PR-Auditor-2.0)

### CODE REVIEW

**Review by craigfurman — APPROVED** [review]
I've not looked under the hood of the new executable being called, and haven't seen this repo much. Not sure who has context on this, so am approving to unblock 🙏

**Comment by BolajiOlajide** [comment]
> I've not looked under the hood of the new executable being called, and haven't seen this repo much. Not sure who has context on this, so am approving to unblock 🙏

It's under the purview of the dev infra team.

**Comment by craigfurman** [comment]
Oh sure, but you requested release team review and the bot assigned me... maybe a typo?

**Pull request merged**
